res/text/message_bank_pokemon_summary_screen.gmm

```diff
@@ -5,27 +5,27 @@
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">{STRVAR_1 1, 0, 0}</language>
 	</row>
-	<row id="pl_msg_00000455_00001" index="1">
+	<row id="pss_male_symbol" index="1">
```

Comment:
**thought**: Is it possible that `PSS` can be confused as an acronym for `Pokemon Storage System`?

Reply:
Remember to also change the prefixes of constants in `include/applications/pokemon_summary_screen/main.h` to not use `PSS`.

Agreed that the prefix `pokemon` is redundant. Another alternative is `summary_menu` (see https://github.com/pret/pokeplatinum/pull/292#issuecomment-2456143448)

Reply:
My gut feeling is that `summary_screen` is more clear/familiar within the specific context of Pokemon than `summary_menu`. I don't mind keeping the `pokemon_` prefix for the organization folder itself. My point here was more that we might want to choose a different acronym, since `PSS` could be associated with a totally different concept.

Reply:
Per discussion in discord, replaced all instances of `PSS` initialism with `SUMMARY`. At a glance none of the names look too egregiously verbose, but I'll revisit them for length in the next PR.